admin_guide/high_availability.adoc

```diff
@@ -10,50 +10,163 @@
+DNS entry, or IP addresses), this could impose operational requirements to
+handle failure cases when one or more router instances are "unreachable".
 
-=== Why IP Failover?
-Within a cluster, the Kubernetes replication controller ensures that the deployment requirements - in particular the number of replicas - are satisfied when the appropriate resources are available.
+For some IP-based traffic services, Virtual IP addresses (VIPs) should always be
+serviced for as long as a single instance is available. This simplifies the
+operational overhead and handles failure cases gracefully.
 
-The link:../architecture/core_objects/routing.html[Router] is one such deployment which can be run with 2 or more replicas to provide a highly available service resilient to individual router instance failures. Depending on how these router instances are discovered (for example via a well-published service or a DNS entry or even IP addresses), this could well impose operational requirements to handle failure cases when 1 or more router instances are "unreachable".
+IMPORTANT: Even though
+a service is highly available, performance can still be affected.
 
-For some IP based traffic services, if we can ensure that the IP addresses are always serviced as long as a single instance is available, this simplifies the operational overhead and handles failure cases gracefully. It is pertinent to note here, that even though the service is highly available, performance could still be impacted.
+Use cases for high-availability include:
 
-=== Use Cases
-  1. As an administrator, I want my cluster to be assigned a resource set
-     and I want the cluster to automatically manage those resources.
-  2. As an administrator, I want my cluster to be assigned a set of virtual
-     IP addresses that the cluster manages and migrates (with zero or
-     minimal downtime) on failure conditions.
-  3. As an addendum to use case #2, the administrator should not be
-     required to perform any manual interaction to update the upstream
-     "discovery" sources (e.g. DNS). The cluster should service all the
-     assigned virtual IPs when atleast a single node is available - and
-     this should be inspite of the fact that the current available
-     resources are not sufficient to reach "critical mass" aka the
-     desired state.
+* As an administrator, I want my cluster to be assigned a resource set and I want the cluster to automatically manage those resources.
+* As an administrator, I want my cluster to be assigned a set of VIPs that the cluster manages and migrates (with zero or minimal downtime) on failure conditions. As an administrator, I should not be required to perform any manual interactions to update the upstream "discovery" sources (e.g. DNS). The cluster should service all the assigned VIPs when at least a single node is available, despite the current available resources not being sufficient to reach the desired state.
```

Comment:
Are all the "as an administrator"s necessary? Both because the audience in the Admin Guide is already assumed to be administrators, and it gets repetitive after three repeats here. I also notice these were split into 3 items in the original, but 2 and 3 were merged here. Intentional?